include/libopencm3/stm32/common/pwr_common_l4.h

```diff
@@ -0,0 +1,174 @@
+/** @addtogroup pwr_defines PWR Defines
+ *
+ * @ingroup STM32L4xx_defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32L4xx Power Control</b>
```

Comment:
What I've started doing with EFM32 is to not include the partname here :) just say "for the Power Control module"